pymatgen/io/openmm/tests/test_inputs.py

```diff
@@ -0,0 +1,313 @@
+# base python
+import pytest
+import tempfile
+
+# cheminformatics
+import numpy as np
+import parmed
+
+# openff
+import openff.toolkit.topology
+from openff.toolkit.typing.engines import smirnoff
+
+# openmm
+import openmm
+from openmm.unit import *
+from openmm import NonbondedForce
+
+# pymatgen
+import pymatgen
+from pymatgen.io.openmm.inputs import (
+    TopologyInput,
+    SystemInput,
+    IntegratorInput,
+    StateInput,
+    OpenMMSet,
+    OpenMMGenerator,
+)
+from pymatgen.io.openmm.tests.datafiles import (
+    input_set_path,
+    topology_path,
+    state_path,
+    corrupted_state_path,
+    integrator_path,
+    system_path,
+    CCO_xyz,
+    CCO_charges,
+    FEC_r_xyz,
+    FEC_s_xyz,
+    FEC_charges,
+    PF6_xyz,
+    PF6_charges,
+)
+
+
+class TestInputFiles:
+    def test_topology_input(self):
+        topology_input = TopologyInput.from_file(topology_path)
+        with tempfile.NamedTemporaryFile() as f:
+            topology_input.write_file(f.name)
+        topology = topology_input.get_topology()
+        assert isinstance(topology, openmm.app.Topology)
+        assert topology.getNumAtoms() == 780
+        assert topology.getNumResidues() == 220
+        assert topology.getNumBonds() == 560
```

Comment:
For each of the tests in `TestInputFiles`, can you add a line to reconstruct the object from the file you wrote? e.g. `topology = TopologyInput.from_file(f)`

Even though the test begins with `from_file`, it doesn't actually confirm that what is written by `write_file` is identical the file in `test_files`.  Adding this one line and then continuing with the test would confirm this.

Reply:
Done. Is what I implemented what you had in mind?